src/docs/product/relay/operating-guidelines.mdx

```diff
@@ -169,6 +169,7 @@ Proxies often set a default maximum body size for requests. Especially native cr
 Internally, Relay makes requests to the configured upstream to forward data and retrieve project configuration. We **highly recommend against** constraining these requests. At present, Relay makes requests to the following endpoints for basic operation:
 
 - All of the above endpoints
+- `/api/0/relays/outcomes/`
```

Comment:
Wait, the outcomes endpoint is still only for internal Relays, right? External relays send their outcomes as client reports.
https://github.com/getsentry/relay/blob/ea87ce6f54aa31cfe49b3d85cb7565a7dc06f588/relay-server/src/endpoints/outcomes.rs#L10

Reply:
You're right, my bad. Closing this PR then.